Cargo.toml

```diff
@@ -37,6 +38,10 @@ features = [
 default = ["std"]
 std = ["ttf-parser/std"]
 
+# experimental features live here.
+experimental = ["wasm_shaper"]
```

Comment:
I do not think we need the `experimental` feature. `wasm_shaper` is more than enough.
And I would prefer dash, i.e. `wasm-shaper`